app/src/main/java/com/rockthevote/grommet/ui/registration/review/ReviewData.kt

```diff
@@ -0,0 +1,24 @@
+package com.rockthevote.grommet.ui.registration.review
+
+data class ReviewData(
+    val hasReviewedAndConfirmedInfo: Boolean,
+    val signature: ByteArray
+) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
what's the reason for the compare methods?

Reply:
Data classes have some limitations accurately conducting an equals check on elements with collections such as ByteArray so it's recommended to override equals and hashCode in these classes.